src/DistTime.java

```diff
@@ -1,312 +1,28 @@
-import javax.swing.*;
-import java.io.*;
 import java.util.*;
 
 public class DistTime {
 
-	private String name, distTime;
 	private double channelRate;
 	private double packetTime;
-	private int numberPackets, burstSize;
-	private double minRate, maxRate, incRate, avgRate, dvRate;
+	private int numberOfPackets;
 	private double ipRate;
-	private double ipRateOn;
 
-	public DistTime(int flitWidth, int flitClockCycles, double Frequency,
-			int PacketSize, int NumberPackets, double Rate) {
-
-		channelRate = Frequency * flitWidth;
-		packetTime = PacketSize * flitClockCycles;
-		numberPackets = NumberPackets;
-		ipRate = (Rate / 100) * channelRate;
-		// ipRate = Rate;
-
-	}
-
-	public DistTime(int flitWidth, int flitClockCycles, String distTime,
-			double frequency, int packetSize, int numberPackets, int burstSize,
-			double avgRate, double minRate, double maxRate, double dvRate,
-			double incRate) {
-		name = "Standard";
-		this.distTime = distTime;
+	public DistTime(int flitWidth, int flitClockCycles, double frequency, int packetSize, int numOfPackets, double rate) {
 		channelRate = frequency * flitWidth;
 		packetTime = packetSize * flitClockCycles;
-		this.numberPackets = numberPackets;
-		this.burstSize = burstSize;
-		this.avgRate = avgRate;
-		this.minRate = minRate;
-		this.maxRate = maxRate;
-		this.dvRate = dvRate;
-		this.incRate = incRate;
-	}
-
-	public ArrayList<String> defineTime() {
-		if (distTime.equalsIgnoreCase("UNIFORM"))
-			return uniform();
-		else if (distTime.equalsIgnoreCase("NORMAL"))
-			return normal();
-		else
-			return pareto();
+		numberOfPackets = numOfPackets;
+		ipRate = (rate / 100) * channelRate;
 	}
 
 	public ArrayList<String> uniform() {
-
-		double idleTime = (double) ((channelRate / ipRate - 1) * packetTime);
+		double idleTime = ((channelRate / ipRate - 1) * packetTime);
 		double totalTime = Math.rint(idleTime) + packetTime;
 		double timestamp = 1;
 		ArrayList<String> vet = new ArrayList<String>();
-
-		for (int j = 0; j < numberPackets; j++) {
+		for (int j = 0; j < numberOfPackets; j++) {
 			vet.add(Integer.toHexString((int) timestamp).toUpperCase());
 			timestamp = timestamp + totalTime;
 		}
 		return vet;
 	}
-
-	public double on(double r, int fator) {
```

Comment:
You should keep the methods: on, off, pareto and normal. These are other kinds of temporal distributions and can be useful.